fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -56,12 +62,45 @@ import org.mozilla.fenix.theme.FirefoxTheme
 @Composable
 fun TabsTrayBanner(
     isInMultiSelectMode: Boolean,
+    selectedTabs: Int,
     selectedPage: Page,
     normalTabCount: Int,
     onTabPageIndicatorClicked: (Page) -> Unit,
 ) {
     if (isInMultiSelectMode) {
-        MultiSelectBanner()
+        val menuItems = mutableListOf(
+            MenuItem(
+                title = stringResource(R.string.tab_tray_multiselect_menu_item_bookmark),
+            ) {},
+            MenuItem(
+                title = stringResource(R.string.tab_tray_multiselect_menu_item_close),
+            ) {},
+        )
+
+        if (Config.channel.isDebug || components.settings.showSecretDebugMenuThisSession) {
```

Comment:
any reference to `components` breaks previews, unfortunately. Let's instead pass-in the boolean directly.